fritz

```diff
@@ -1,9 +1,86 @@
 #!/usr/bin/env python
 import argparse
-import sys
-import subprocess
+from contextlib import contextmanager
+import json
 import os
 from pathlib import Path
+import subprocess
+import sys
+
+
+@contextmanager
+def status(message):
+    # stolen from baselayer
+    print(f'[·] {message}', end='')
+    try:
+        yield
+    except:
+        print(f'\r[✗] {message}')
+        raise
+    else:
+        print(f'\r[✓] {message}')
+
+
+def check_configs(cfgs):
```

Comment:
We can just make empty configuration files if they don't exist, then they won't override the defaults.

Reply:
Kowalski would be expecting values for some settings, so empty ones would raise errors. I think SkyPortal also didn't like empty db pwds or something. Can you elaborate on your point, I may be misunderstanding something?